engines/order_management/app/services/order_management/reports/bulk_coop/bulk_coop_report.rb

```diff
@@ -171,7 +171,7 @@ def customer_payments_amount_owed(line_items)
           if OpenFoodNetwork::FeatureToggle.enabled?(:customer_balance, @user)
             unique_orders(line_items).sum(&:new_outstanding_balance)
           else
-            unique_orders(line_items).sum(&:outstanding_balance)
+            unique_orders(line_items).sum(&:old_outstanding_balance)
```

Comment:
The PR headline is "Feature-toggle all remaining parts of the app where the order balance is shown" but this report is still showing the old balance, regardless of the feature-toggle, right?

Reply:
nope, look at the `if` above. This report was tackled before all this work and I don't think it's worth refactoring it again. It's been thoroughly tested and it's proven to work with the feature toggle.

Reply:
Ha, of course! Sorry. :see_no_evil: